assignments/closure.js

```diff
@@ -1,20 +1,100 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
+function animalSpeak(animalSound) {
```

Comment:
Way to go above and beyond on these next challenges. You could have done a simple function but I appreciate the extra work (I really like this animalSpeak function too, just fun).